news/2018-06-06-beatmap-spotlights-march-and-april-2018.md

```diff
@@ -1,12 +1,12 @@
 ---
 layout: post
 title: "Beatmap Spotlights: March & April 2018"
-date: 2018-06-06 06:00:00 +0000
```

Comment:
this file should have no changes. looks like a global find/replace mistake

Reply:
What is this 👀 I have the correct data on these fields